ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1460,6 +1735,24 @@
 /* End PBXHeadersBuildPhase section */
 
 /* Begin PBXNativeTarget section */
+		A9B315531B3940610001CB9C /* ReactiveCocoa-watchOS */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = A9B3155D1B3940610001CB9C /* Build configuration list for PBXNativeTarget "ReactiveCocoa-watchOS" */;
+			buildPhases = (
+				A9B3154F1B3940610001CB9C /* Sources */,
+				A9B315501B3940610001CB9C /* Frameworks */,
+				A9B315511B3940610001CB9C /* Headers */,
+				A9B315521B3940610001CB9C /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+			);
+			name = "ReactiveCocoa-watchOS";
+			productName = "ReactiveCocoa-watchOS";
```

Comment:
This should be `ReactiveCocoa` so that framework is named correctly. Generally I set this to `$(PROJECT_NAME)` for multi-target frameworks.